### ISSUE TITLE
MARC 022 $2

### ISSUE THREAD

**Issue by kiegel** [lcnetdev/marc2bibframe#218]
Subfield 2 of field 022 (ISSN) contains a code for the source of the ISSN.

022	0_ |a 0392-4041 |l 0392-4041 |2 d

becomes

~~~xml
bf:issn [ a bf:Identifier ;
            bf:identifierAssigner "d" ;
            bf:identifierScheme "issn" ;
            bf:identifierValue "0392-4041" ] ;
~~~

The assigner identity "d" is not useful.  It would be helpful if the ISSN National Centres code list were converted to a thesaurus and URIs could be assigned instead.

In that case, changes would be needed to the BF property.  bf:identifierAssigner currently has a range of Literal.  To follow the usual pattern, it would make sense for bf:identifierAssigner to have a range of URIs, while a new property such as bf:identifierAssignerNote had a range of Literal.

(OCLC # 56092311)

**Comment by ntra00** [comment]
Great idea. I don't have a list of issn assigners in RDF; do you know of one?

**Comment by kiegel** [comment]
No, we don't.  We thought that LC could nudge the ISSN center to create one, or have LC host it for them.

**Comment by ntra00** [comment]
Here is the list, but I don't see alpha characters, just numbers. Am I in the wrong place?
http://www.issn.org/wp-content/uploads/2013/09/ENG_Codes_pays_ISO_codes_centres_ISSN-20140102.pdf

**Comment by kiegel** [comment]
There are some letters in the list.  For example, Italy is "d", which is the value in the sample record we used.

**Comment by ntra00** [comment]
Oops I missed that. thanks. I spoke with our ISSN ctr rep, and she's moving the conversation along with Paris. I hope not to host it ourselves, since we don't want to duplicate their maintenance. On a brighter note, I did post subjectSchemes in ID, for 65X $2. We do want more of these.
